doc/backend_vars.asciidoc

```diff
@@ -65,6 +65,7 @@ TAPDEV;device name;undef;TAP device name to which virtual NIC should be connecte
 TAPSCRIPT;;;
 TESTDEBUG;boolean;0;Enable test debugging: override 'milestone' and 'fatal' test flags to 1. Snapshot are created after each successful test module and each fail aborts test run
 UEFI;;;
+UEFI_PFLASH;;;
```

Comment:
the point of this file is to document the meaning of the variables. So it's your chance to add a description here.